### PR TITLE
Add Fedora 22 setup.

### PR DESCRIPTION
**Pull request by imtayadeway**
* fedora's package manager has changed from yum -> dnf
* nokogiri requires that the patch package be installed separately

### CODE REVIEW

**Comment by jrafanie** [comment]
:heart:  Thanks!

@movitto @brandondunne @martinpovolny Please review

File: developer_setup.md [hunk 1]
```diff
@@ -31,6 +31,35 @@
...
+  ```
+
+* Configure PostgreSQL
+
+  ```bash
+  sudo passwd postgres <new_password>
+  su postgres -c 'initdb -D /var/lib/pgsql/data'
+  sudo systemctl enable postgresql
+  sudo systemctl start postgresql
+  su postgres -c "psql -c \"CREATE ROLE root SUPERUSER LOGIN PASSWORD 'smartvm'\""
```
Comment (Fryguy) [inline developer_setup.md]: This looks identical to Fedora 20 section...can we just update that section title to be Fedora 20+.  Unless I'm missing something.

Comment (imtayadeway) [inline developer_setup.md]: yep it is identical - i'll adjust as above

**Comment by imtayadeway** [comment]
@Fryguy i just removed the redunancy and added an extra line for clarity. i'll squash if you think this is OK

**Comment by movitto** [comment]
@jrafanie I run miq off the appliance and haven't set it up on fedora in a while so I can't comment on the specific steps. Generally speaking it looks fine though.

I wonder if the app & underlying stack could simply be yum installed at some point. That + an installer could simplify the setup experience.

**Comment by brandondunne** [comment]
:+1: LGTM

**Comment by jrafanie** [comment]
@movitto rpms aren't cross platform so that makes it harder.  If we used a platform independent packaging format, maybe but it feels like a lot of overhead for what bundler does already for at least 90% of our stack.  

More appliance formats + container support could simplify the process provided each platform makes it easy to setup the hypervisor/docker environment.  If you want to help out in those areas, we can certainly use it!  :heart:

**Comment by movitto** [comment]
@jrafanie appliances / containers would still be limited to the environments we support & would still entail building the multiple environments as you mentioned.

Packages are fairly platform specific but still are needed & provide tradeoffs w/ a bundler centric solution. An alternate solution would be to build packages for multiple platforms as needed.

**Comment by jrafanie** [comment]
Yes, dependencies are hard :cry: especially when we have platform specific solutions to work with.

**Comment by Fryguy** [comment]
To @movitto's point, would it make sense to make some kind of meta-rpm that has all the dependencies for Fedora?  Like a `manageiq-devel` rpm that just has dependencies on those other devel packages?

**Comment by Fryguy** [comment]
Also, I just noticed that the package manager is `dnf` and not `yum`. **That** was the difference.  When did they change that?  Is `yum` even available anymore?

**Comment by imtayadeway** [comment]
@Fryguy you can still use `yum` commands in 22 but it will complain and issue the `dnf` command for you instead.

**Comment by jrafanie** [comment]
:+1: Let's make any further changes as we find issues.  Thanks @imtayadeway

**Pull request merged**